stakepooldclient/stakepooldclient.go

```diff
@@ -352,3 +352,29 @@ func (s *StakepooldManager) RPCStatus() []string {
 
 	return stakepooldPageInfo
 }
+
+// MyMasterPubKeys performs gRPC MasterPubKeys requests against all stakepoold
+// instances and returns all results fetched without errors. Returns a map of keys
+// to accounts or an error if no keys were found.
+func (s *StakepooldManager) MyMasterPubKeys() (map[string]string, error) {
+	pubkeymap := make(map[string]string)
+	for i, conn := range s.grpcConnections {
+		client := pb.NewStakepooldServiceClient(conn)
+		resp, err := client.MyMasterPubKeys(context.Background(), &pb.MyMasterPubKeysRequest{})
+		if err != nil {
+			log.Warnf("GetMasterPubkeys gRPC failed on stakepoold instance %d: %v", i, err)
+			continue
+		}
+		for _, pubkey := range resp.MasterPubKeys {
+			pubkeymap[pubkey.PubKey] = pubkey.Account
+		}
+	}
+	if len(pubkeymap) == 0 {
```

Comment:
I dont think we need this check because afaik this condition should be impossible. Every instance of dcrwallet will always have at least one MasterPubKey because every instance of dcrwallet has a `default` account. Can anybody else confirm?

Reply:
the requests don't return error if they err on rpc calls, so this was to err out if the children didn't find anything. It would be better to have this behavior in the children to be more clear I guess.

Reply:
behavior moved to children, stakepoold instances